amass/sources/sources.go

```diff
@@ -20,15 +20,22 @@ import (
 )
 
 var (
-	nameStripRE = regexp.MustCompile("^((20)|(25)|(2f)|(3d)|(40))+")
+	nameStripRE = regexp.MustCompile("^((20)|(25)|(2b)|(2f)|(3d)|(3a)|(40))+")
 )
 
 // Clean up the names scraped from the web.
 func cleanName(name string) string {
-	if i := nameStripRE.FindStringIndex(name); i != nil {
-		name = name[i[1]:]
-	}
 	name = strings.TrimSpace(strings.ToLower(name))
+
+	for {
+		if i := nameStripRE.FindStringIndex(name); i != nil {
+			name = name[i[1]:]
+		} else {
+			break
+		}
+	}
```

Comment:
This seems pretty complex function. Do you have a unit test covering it?

Reply:
You're right. There's no unit test for it yet, so we'll need to add one. For now though, these fixes need to be made available to users